modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,6 +28,31 @@ Older versions may work, but are not supported.
 
 The most current and up to date API Documentation is always available through the https://pkg.go.dev/github.com/couchbase/gocb/v2[godoc website].
 
+== Version 2.3.4 (16 November 2021)
+
+Version 2.3.4 is a maintenance release for the Go SDK 2.3.0.
+
+https://pkg.go.dev/github.com/couchbase/gocb/v2@v2.3.4?tab=doc[API Documentation]
```

Comment:
Currently 404

Reply:
Try again, think the pkg.go.dev index just needed poking.